prepare.ps1

```diff
@@ -782,7 +766,7 @@ Set-ItemProperty -Path $path -Name 'CPUOverclocking' -Value '10'
 # 
 # 17. Add in a game art scraper
 # 
-$scraperZip = $requirementsFolder + "scraper_windows_amd64*.zip"
+$scraperZip = $requirementsFolder + "scraper_windows_amd64-v1.4.5.zip"
```

Comment:
Isn't this the incorrect version?

Reply:
The correct version is the one with the *. You'll have to exclude this PR because it was made before the other one.